packages/block-editor/src/components/panel-color-settings/README.md

```diff
@@ -0,0 +1,42 @@
+# `PanelColorSettings`
+
+Render a panel with color choices shown in circles, allowing a user to pick them.
+
+## Properties
+
+### `children`
+
+* **Type:** `Element`
+* *Required:*  No
+* *Default:* Null
+
+The content to be displayed within the panel.
+
+### `colors`
+
+Array with the colors to be shown.
+
+* *Type:* `Array`
+* *Required:* Yes
+
+### `colorSettings`
+
+* **Type:** `Array`
+* *Required:* Yes
+
+An array of props used for mapping both ColorIndicator and ColorPaletteControl components.
```

Comment:
I think it might be good to mention that each entry in the array represents an individual color palette. An example showing the shape of the object might be good here as well.